### PR TITLE
[RW-4254][RISK=NO] Wording changes to Select All communication

### PR DESCRIPTION
**Pull request by NehaBroad**
- [x ] This PR meets the Acceptance Criteria in the JIRA story
- [ x] The JIRA story has been moved to Dev Review
- [ x] This PR includes appropriate unit tests
- [ x] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

Screen shots:

Screen Shot 2020-01-28 at 10 25 49 AM
Screen Shot 2020-01-28 at 10 25 55 AM

### CODE REVIEW

File: ui/src/app/pages/data/concept/concept-table.tsx [hunk 1]
```diff
@@ -263,15 +263,18 @@ export class ConceptTable extends React.Component<Props, State> {
...
 
   selectAllHeader() {
-    if (this.state.showBanner) {
-      const {concepts} = this.props;
-      const bannerText = this.state.selectAll ? 'Clear Selection' : 'Select ' + concepts.length;
-      return <FlexRow  data-test-id='selection' style={{fontWeight: '200'}}>
-        All concepts on this page are selected.&nbsp;
+    const {concepts} = this.props;
+    if (this.state.showBanner && concepts.length > ROWS_TO_DISPLAY) {
+      const selectedConceptSize = this.state.selectAll ? concepts.length : ROWS_TO_DISPLAY;
```
Comment (calbach) [inline ui/src/app/pages/data/concept/concept-table.tsx]: I'm confused by this. At this point in the code is there some kind of guarantee that the user has selected everything being displayed on the page? If `selectAll` is false, why do we assume they have selected `ROWS_TO_DISPLAY` concepts.

File: ui/src/app/pages/data/concept/concept-table.tsx [hunk 2]
```diff
@@ -263,15 +263,18 @@ export class ConceptTable extends React.Component<Props, State> {
...
+      const clickableText = this.state.selectAll ? 'Clear Selection' :
+          'Select all ' + concepts.length + ' concepts';
+      return <div data-test-id='selection'><FlexRow style={{fontWeight: '200'}}>
+        You’ve selected all {selectedConceptSize} concepts. &nbsp;
```
Comment (calbach) [inline ui/src/app/pages/data/concept/concept-table.tsx]: The word "all" here does not seem like it should be included in the case where they haven't selected everything.

File: ui/src/app/pages/data/concept/concept-table.tsx (hunk 1 above)
Comment (NehaBroad) [reply ui/src/app/pages/data/concept/concept-table.tsx]: the method updateSelectedConceptList(selectedConcepts, origin) sets the state variable showBanner to be true only if the user selects the checkbox in the header

File: ui/src/app/pages/data/concept/concept-table.tsx (hunk 2 above)
Comment (NehaBroad) [reply ui/src/app/pages/data/concept/concept-table.tsx]: I missed the text, its supposed to be You’ve selected all 20 concepts **on this page**.

File: ui/src/app/pages/data/concept/concept-table.tsx (hunk 1 above)
Comment (calbach) [reply ui/src/app/pages/data/concept/concept-table.tsx]: I think `showBanner` should be renamed to something like `allVisibleConceptsSelected`. Also, I can't reconcile the logic at L204 with what you're saying here.

Say I do the following - doesn't this work incorrectly?
1. Click select all checkbox
-> banner says "selected 20 concepts"
2. deselect one concept
 -> banner still says "selected 20 concepts"?

**Pull request merged**